scripts/build.js

```diff
@@ -167,15 +203,23 @@ program
     '-w, --workspace [workspace]',
     'Scope build to a specific workspace, its packages and their dependencies',
   )
+  .option('--verbose', 'output extra debugging')
   .action(async () => {
+    process.env.VERBOSE = program.verbose;
```

Comment:
why does it need to be a `env` variable ?

Reply:
Environment variable need to be passed to the Worker Thread via [`SHARE_ENV`](https://nodejs.org/api/worker_threads.html#worker_threads_worker_share_env).

Used here: https://github.com/ovh/manager/pull/3328/files#diff-a992dd64a8c626fedc95d613f43602b91f6dc6339893afaf405dc05fa13c9ceaR150

It helps us to output extra logs if needed.